atomic_reactor/plugins/cachi2_init.py

```diff
@@ -135,6 +135,7 @@ def process_remote_sources(self) -> List[Dict[str, Any]]:
                 remote_source_data["ref"]
             )
 
+            enforce_sandbox(source_path_app, remove_unsafe_symlinks=False)
```

Comment:
will be removal of unsafe symlinks implemented in followup PR?

Reply:
I had intended it for this PR, but it can be in a followup if you want to get this merged sooner.